cypress/support/commands.js

```diff
@@ -61,15 +61,6 @@ Cypress.Commands.add('ExactMatch', { prevSubject: 'element' }, (elements, expect
   throw `Exact Match '${expectedText}' NOT Found`
 })
 
```

Comment:
I suspect this code's value is going to be more apparent as compound Actions are brought online. I'd strongly suggest not removing the functionality

Reply:
I assume you are refering to the ExactMatchComplexHTML that you wrote and I deleted. While it was in many ways an elegant solution, it was not the Cypress way. To fully understand why you need to look at the function that called it which is `ClickEndSessionAction` in ScorerModel.js.

Each `.chainFunction()` call in that code is actually a verification that the element exists, and if it fails, Cypress retries up to the time out period. Also each one of those `.chainFunction()` logs to the left pane in the browser in which the test is run, thus you can pinpoint what expected element was found or not and determine the cause of failure.

Cypress is a complex beast, but it does a lot of cools stuff for us and helps document each step in a test scenario pretty well. We should only deviated from their canned functions when we need to and not when the Cypress patterns will work.

To that end, I don't see that we would ever need that function.